cmd/oci-unpack/main.go

```diff
@@ -101,6 +101,11 @@ func (v *unpackCmd) Run(cmd *cobra.Command, args []string) {
 		os.Exit(1)
 	}
 
+	if _, err := os.Stat(args[1]); os.IsNotExist(err) {
+		v.stderr.Printf("destination path %s does not exist", args[1])
+		os.Exit(1)
```

Comment:
It looks like the current code goes with `MkdirAll` as part of the [layer][1] [unpacking][2].  I think we probably want to transition that to a `Mkdir` approach for the `dest` directory, but I agree that the error floated here in 7e059a6 is too strict.

[1]: https://github.com/opencontainers/image-tools/blob/27d5fd6616cc7cd4de36521e9a0c9c8cfc8234ca/image/manifest.go#L169
[2]: https://github.com/opencontainers/image-tools/blob/27d5fd6616cc7cd4de36521e9a0c9c8cfc8234ca/image/manifest.go#L201